packages/wallet-adapter-react/src/WalletProvider.tsx

```diff
@@ -57,7 +57,7 @@ export const AptosWalletAdapterProvider: FC<AptosWalletProviderProps> = ({
   // https://github.com/aptos-labs/aptos-wallet-adapter/issues/94
   const [isLoading, setIsLoading] = useState<boolean>(true);
 
-  const walletCore = useMemo(() => new WalletCore(plugins), []);
+  const walletCore = useMemo(() => new WalletCore(plugins ?? []), []);
```

Comment:
Shouldn't this have `plugins` in the dependency array?
Or is that intended

Reply:
this is to support only the new standard - meaning if a dapp wants to use then ew standard (and not the legacy one) they should not pass in any wallet plugin

Reply:
I just re-read you comment. I don think having it in the dependency array is needed as this array is being passed one time when using the adapter react provider.